packages/cli/src/cmds/preview/preview.ts

```diff
@@ -136,7 +140,7 @@ export default class Preview extends YargsCommand {
       const result =
         previewType === "local"
           ? await this.localPreview(workspaceFolder, browser)
-          : await this.remotePreview(workspaceFolder, browser);
+          : await this.remotePreview(workspaceFolder, browser, args.env as any);
```

Comment:
What if `env` is undefined?

Reply:
The same as above, the user needs to choose one.